apps/linkedin-to-notion/src/background/index.ts

```diff
@@ -8,9 +8,12 @@ chrome.action.onClicked.addListener((tab) => {
   }
 });
 
-chrome.tabs.onUpdated.addListener((tabId, _, tab) => {
-  if (tab.url.match(linkedInURLRegex)) {
-    return chrome.tabs.sendMessage(tabId, 'updateLinkedInNotionSidePanel');
+chrome.tabs.onUpdated.addListener((tabId, { status }, tab) => {
+  if (tab.url?.match(linkedInURLRegex)) {
+    if (status === 'complete') {
+      chrome.tabs.sendMessage(tabId, 'updateLinkedInNotionSidePanel');
+    }
+  } else {
+    chrome.tabs.sendMessage(tabId, 'closeLinkedInNotionSidePanel');
```

Comment:
I think we should start by awaiting those messages, it would be cleaner
```
chrome.tabs.onUpdated.addListener(async (tabId, { status }, tab) => {
  if (tab.url?.match(linkedInURLRegex)) {
    if (status === 'complete') {
      await chrome.tabs.sendMessage(tabId, 'updateLinkedInNotionSidePanel');
    }
  } else {
    await chrome.tabs.sendMessage(tabId, 'closeLinkedInNotionSidePanel');
```

Reply:
cf. comment above + awaiting messages:
```
chrome.tabs.onUpdated.addListener(async (tabId, { status }, tab) => {
  if (tab.url && tab.url.match(linkedInURLRegex)) {
    if (status === 'complete') {
      return setTimeout(() => chrome.tabs.sendMessage(tabId, 'updateLinkedInNotionSidePanel'), 2000);
    }
  } else {
    return await chrome.tabs.sendMessage(tabId, 'closeLinkedInNotionSidePanel');
  }
```

Reply:
Ok for me but add a comment with a TODO to remind us to fix this 👌